### DISCUSSION TITLE
Non-exempt encryption flag

### DISCUSSION THREAD

**Discussion by saidbyced** [SwiftPackageIndex/SwiftPackageIndex-Server#3114] [Ideas]
Something I've been looking at is Apple's requirements on [Complying with Encryption Export Regulations](https://developer.apple.com/documentation/security/complying_with_encryption_export_regulations). Since it asks to include all third-party SDKs, I was wondering if there could be a flag in the header if something either doesn't declare, or declares non-exempt encryption, in a plist. Is something like this possible?
![3150498~dark@2x](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/assets/23424740/2bb3862e-2784-4926-9404-e835530c27b0)

**Comment by heckj** [comment]
That declaration is only in the application's Info.plist, and not represented in the Package.swift manifest of the package. As such, it's not available - and not declared - such that the Swift Package Index could capture it.

**Reply by finestructure** [reply]
Yeah, I think we'd need to have this in a predictable location, ideally the manifest, in order to show it.

We could look for `Info.plist` files throughout a repo but that can easily get out of hand by picking up files from subdirectories, like example projects or git submodules.
